src/Provider/Provider.Web/Controllers/VacancyPreviewController.cs

```diff
@@ -46,6 +46,14 @@ public async Task<IActionResult> VacancyPreview(VacancyRouteModel vrm)
         [Authorize(Policy = nameof(PolicyNames.HasContributorOrAbovePermission))]
         public async Task<IActionResult> Submit(SubmitEditModel m)
         {
+            if (!ModelState.IsValid)
+            {
+                var viewModel1 = await _orchestrator.GetVacancyPreviewViewModelAsync(m);
```

Comment:
Same as comment above about using single variable

Reply:
Amended in latest.